src/components/CheckListContainer.test.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import ChecklistContainer from "./CheckListContainer";
+
+const data = [
+  {
+    text: "Go to Checklist",
+    title: "Cat Adoption Checklist",
+    link: "denemeLink",
+  },
+  {
+    text: "Go to Checklist",
+    title: "Cat Adoption Checklist",
+    link: "denemeLink",
+  },
+  {
+    text: "Go to Checklist",
+    title: "Cat Adoption Checklist",
+    link: "denemeLink",
+  },
+];
+
+it("renders correctly", () => {
```

Comment:
can we change the test name to be something related to the test it self, maybe smth like checklist container renders correctly ?

Reply:
Sure. Thanks Halit